.github/PULL_REQUEST_TEMPLATE/api.md

```diff
@@ -0,0 +1,16 @@
+<!--
+IMPORTANT:
+If your PR doesn't close a particular issue, please, create the issue first and describe the whole context: what you're adding/changing and why you're doing so. And only then open the Pull Request, which would close that issue!
+-->
+
+## Issue
+
+Closes issue #number.
+
+## Checklist
+
+- [ ] I have updated CHANGELOG.md
+- [ ] This part of API is exported from `src/index.ts`
```

Comment:
I'm a bit skeptical about that kind of export.

Perhaps it would be better to create a few `index.ts` files inside different packages to avoid name clashes and make the meaning of the specific import clearer for the developer. For example, I would definitely suggest having separate `grammar/index.ts` and `func/index.ts` imports, because they have the same function and type names.

Thus, I would suggest changing it like this:

```suggestion
- [ ] This part of API is exported from `index.ts`
```

Reply:
Creating `index.ts` files in each folder inside `src/` where each one simply exports all the contents of other files is a common pattern in JS/TS world for hacking together directory-scoped imports :)

So while I agree about starting to follow the `index.ts` pattern, the actual line in the PR here probably meant the global, publicly exposed API elements, if I got @anton-trunov idea right. Thus, `src/index.ts` probably has to stay as is, and all collisions into it may be resolved by import aliases: `import { parse as parseTact } from ...`, for example

Reply:
@jubnzv Sure, the multiple exports idea sounds better. Here is the corresponding issue: https://github.com/tact-lang/tact/issues/740

Reply:
I just slightly changed the wording here, I hope it's fine